R/readGlobal_Steel_Plant_Tracker.R

```diff
@@ -0,0 +1,20 @@
+#' Read Global Steel Plant Tracker Data
+#'
+#' Read data from https://globalenergymonitor.org/projects/global-steel-plant-tracker/.
+#'
+#' @return A [`magpie`][magclass::magclass] object.
+#'
+#' @importFrom magrittr %>%
+#' @importFrom quitte madrat_mule
+#' @importFrom readxl read_xlsx
+#'
+#' @export
+readGlobal_Steel_Plant_Tracker <- function() {
```

Comment:
for new functions, please adhere to the convention, i.e `readGlobalSteelPlantTracker